app/main.py

```diff
@@ -1 +1,80 @@
 # write your code here
+class SoftwareEngineer:
+    def __init__(
+        self,
+        name: str,
+    ) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(
+        self,
+        name: str,
+    ) -> None:
+        super().__init__(
+            name=name,
+        )
+        self.skills = [
+            "JavaScript",
+            "HTML",
+            "CSS",
+        ]
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super(BackendDeveloper, self).__init__(
+            name=name
+        )
+        self.skills = [
+            "Python",
+            "SQL",
+            "Django",
+        ]
+
+    def create_powerful_api(self) -> str:
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(
+            name=name,
+        )
+        self.skills = [
+            "Java",
+            "Android studio"
+        ]
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(BackendDeveloper, FrontendDeveloper):
+    def __init__(
+        self,
+        name: str
+    ) -> None:
+        super().__init__(name)
+        self.skills = []
+        for skill in BackendDeveloper(name).skills:
```

Comment:
when you inherit from a class you already get the attributes of the class. it is useless